src/chrome/content/rules/lezhin.com.xml

```diff
@@ -1,19 +1,19 @@
 <ruleset name="lezhin.com">
 	<!--
 	Certificate error:
-	social.lezhin.com
 	help.lezhin.com
-	ncdn.lezhin.com
-	cdn.lezhin.com
 
-	Refused by HTTP 404:
-	desk.lezhin.com
+	Refused by HTTP 403:
 	link.lezhin.com
 	-->
 	
 	<target host="lezhin.com" />
 	<target host="www.lezhin.com" />
 	<target host="m.lezhin.com" />
+	<target host="desk.lezhin.com" />
+	<target host="cdn.lezhin.com" />
+	<target host="acdn.lezhin.com" />
+	<target host="ncdn.lezhin.com" />
```

Comment:
ggtoon.lezhin.com
social.lezhin.com

It seems that there are also mirrors like:
\d+cdn.lezhin.com
\d+www.lezhin.com